include/ofi_net.h

```diff
@@ -404,6 +404,23 @@ static inline ssize_t ofi_byteq_recv(struct ofi_byteq *byteq, SOCKET sock)
 	return ret;
 }
 
+static inline ssize_t ofi_byteq_uring_recv(struct ofi_byteq *byteq,
+					   struct ofi_uring *uring,
+					   struct ofi_uring_ctx *uctx,
+					   SOCKET sock)
+{
+	size_t avail;
+	ssize_t ret;
+
+	avail = ofi_byteq_writeable(byteq);
+	assert(avail);
+	ret = ofi_uring_recv(uring, uctx, sock, &byteq->data[byteq->tail], avail,
+			     MSG_NOSIGNAL);
+	if (ret > 0)
+		ofi_byteq_write_advance(byteq, ret);
+	return ret;
+}
```

Comment:
I need more details in the commit message to understand the approach here.  Is bsock the right place to add this abstraction?  Should this be higher or lower?  Looking at some of the calls, the separation seems to be between using iouring versus socket API.  Are the other pieces of byteq used with iouring?